customer-commons/src/test/resources/features/problem/bad_gateway_template.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I was going to use these template files for the tests, so they're maybe superfluous.